KlipTok.de-DE.json

```diff
@@ -2,43 +2,115 @@
 	// Components
 
 	// Clip
-	"Recorded": "{0} aufgenommen",
+	"Recorded": "{0} aufgenommen", // {0} ist der relative Zeitstempel der Aufnahme: "vor 3 Tagen"
 	"Likes": "Gefällt mir",
 	"Comments": "Kommentare",
 	"ClipViews": "Aufrufe",
-	"ShareThisClip": "Teile diesen Clip",
+	"ShareThisClip": "Diesen Clip teilen",
 	"KlipTokContentPlaceholder": "KlipTok Platzhalter",
 
 	// Comments
-	"ByLineFormat": "{1} hat am {0} geschrieben:",
+	"ByLineFormat": "{1} hat {0} geschrieben:", // {0} ist der relative Zeitstempel des Kommentars, {1} ist der Name des Kommentierenden
 	"Reply": "Antworten",
 
 	// Login View
 	"DarkMode": "Dunkles Design",
-	"LoginWithTwitch": "Einloggen mit Twitch",
+	"LoginWithTwitch": "Mit Twitch Einloggen",
 	"Logout": "Ausloggen",
 
 	// Header
 	"SearchForChannel": "Kanal suchen...",
-	"SearchNotFoundFormat": "{0} nicht gefunden",
+	"SearchNotFoundFormat": "Keine Ergebnisse für {0}", // {0} ist der Suchbegriff, der nicht gefunden wurde
 
 	// Sidebar
 	"SuggestedStreamers": "Vorgeschlagene Streamer",
-	"FollowedStreamers": "Streamer, denen du folgst",
+	"FollowedStreamers": "Streamer, denen ich folge",
 	"AllStreamers": "Alle Streamer",
 	"About": "Über uns",
 	"Close": "Schließen",
 
 	// Streamer Page
-	"ClickToFollow": "Klicken um zu Folgen",
-	"ClickToUnfollow": "Klicken um zu Entfolgen",
-	"ActionOnTwitchTemplate": "{0} auf Twitch",
+	"ClickToFollow": "Auf Twitch Folgen",
+	"ClickToUnfollow": "Auf Twitch Entfolgen",
+	"ActionOnTwitchTemplate": "{0} auf Twitch", // {0} wird ersetzt mit "Follow" oder "Unfollow" -> "Follow on Twitch"
```

Comment:
Will the `(Un-)Follow` tokens be localized as well at some point?

Reply:
I'm not sure what your request is here...  The "Action on Twitch template" adds the action "Follow" or "Unfollow" where the token {0} is

Reply:
Why "Auf Twitch Folgen / Entfolgen" (Follow on Twitch)? Isn't this about Following on KlipTok?

Reply:
@luke-777 If you're referring to lines 33/34, those buttons actually modify your follows on Twitch, which KlipTok then mirrors.

@csharpfritz What I'm asking is basically: To me the comment reads, as if `{0}` is replaced by literals, and not localized strings, which would result in `Follow auf Twitch` (doesn't make much sense). So I assumed `Follow` and `Unfollow` will be tokenized to enable localization on those as well. I might be missing something obvious here, though.

Reply:
No..  The {0} is replaced with the translation of 'ClickToFollow' and 'ClickToUnfollow' from lines 33 & 34